.test-infra/kubernetes/cassandra/LargeITCluster/start-up.sh

```diff
@@ -17,13 +17,6 @@
 #!/bin/bash
 set -e
 
-# Load YCSB tool
-echo "Downloading YCSB tool"
-echo "------------------------------"
-curl -O --location https://github.com/brianfrankcooper/YCSB/releases/download/0.12.0/ycsb-0.12.0.tar.gz
-tar xfz ycsb-0.12.0.tar.gz
-wget https://www.slf4j.org/dist/slf4j-1.7.22.tar.gz
-tar xfz slf4j-1.7.22.tar.gz
-cp slf4j-1.7.22/slf4j-simple-*.jar ycsb-0.12.0/lib/
-cp slf4j-1.7.22/slf4j-api-*.jar ycsb-0.12.0/lib/
-echo "YCSB tool loaded"
```

Comment:
looks like this file got lost in the renames? Should not block merge today, we can add it back in